tests/models/registry.py

```diff
@@ -913,6 +913,10 @@ def check_available_online(
         "Qwen/Qwen2.5-VL-7B-Instruct",
         speculative_model="Rayzl/qwen2.5-vl-7b-eagle3-sgl",
     ),
+    "Eagle3Qwen3vlForCausalLM": _HfExamplesInfo(
```

Comment:
```suggestion
    "Eagle3Qwen3VLForCausalLM": _HfExamplesInfo(
```

Not sure whether this is a typo

Reply:
> Same for the one in the main registry

Thanks for your suggestion. The architecture name "Eagle3Qwen3vlForCausalLM" was adapted from the Qwen2-VL Eagle3 model's original name "Eagle3Qwen2_5vlForCausalLM". While Qwen3-VL's official designation is indeed "Qwen3VLForConditionalGeneration", this difference should not affect normal usage